### PR TITLE
make submenus and make output of add accounts as table

### PR DESCRIPTION
**Pull request by rwobben**
I have made submenus so the new structure is more visible. 

And I made it work that the output of doCreateAccounts will now be this : 

     Added Accounts : 
 
    name                 amount
    cash on hands         10000

### CODE REVIEW

File: paytime/app/Main.hs [hunk 1]
```diff
@@ -11,40 +12,37 @@ main :: IO ()
...
-            "MW"   -> do
-                       newAccounts <- doWithDrawlAccounts accounts
-                       runLoop newAccounts
-            "TB"  -> do
-                       putStrLn $ "Total balance: " ++ show (sumAccounts accounts) 
-                       runLoop accounts
+            "AC" ->do 
+                      let canDelete = length accounts > 0
+                      putStrLn $ ""
+                      putStrLn $ "subMenu Accounts \n"
```
Comment (GetContented) [inline paytime/app/Main.hs]: You don't need ($) in the two lines above.

File: paytime/app/Main.hs [hunk 2]
```diff
@@ -11,40 +12,37 @@ main :: IO ()
...
+                      putStrLn $"CA: Create Accounts.\n" ++ (if canDelete then "\nDA: Delete Account." else "")
+                      putStr $ "Your Choice: "
```
Comment (GetContented) [inline paytime/app/Main.hs]: You don't need ($) here.

File: paytime/src/Lib.hs [hunk 3]
```diff
@@ -49,12 +59,17 @@ doCreateAccounts accts = do
         newBal <- getBalance
         let newAcct     = makeAccount newName newBal
             newAccounts = addAccount newAcct accts
-        putStrLn $ "Account " ++ newName ++ " made with balance " ++ show newBal ++ "."
+        putStrLn " "
+        putStrLn "Added Accounts : "
+        putStrLn " "
+        printTable [ ["name", "amount"], [newName, show(newBal)]]
```
Comment (GetContented) [inline paytime/src/Lib.hs]: you don't need parenthsis around the argument to show.

**Comment by GetContented** [comment]
Looks good other than those comments.

File: paytime/app/Main.hs (hunk 1 above)
Comment (rwobben) [inline paytime/app/Main.hs]: Changed.

File: paytime/src/Lib.hs (hunk 3 above)
Comment (rwobben) [inline paytime/src/Lib.hs]: Changed

**Comment by rwobben** [comment]
:) Then im happy

**Comment by GetContented** [comment]
Like I said before, this looks fine to me.

**Comment by rwobben** [comment]
If you know and can learn how to change this the way you wanted Im open.

**Pull request merged**